test-data-generator/src/test/scala/za/co/absa/spline/testdatagen/ConfigSpec.scala

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2019 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.spline.testdatagen
+
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.matchers.should.Matchers.{all, convertToAnyShouldWrapper}
+
+class ConfigSpec extends AnyFlatSpec{
+
+  private val varAttr = Config(reads = Constant(2) ,operations = Constant(5), attributes = Variable(2,8,2), expressions = Constant(3))
+  private val constTriangle = Config(graphType= "triangle", reads= Constant(3), operations = Constant(1), attributes = Constant(2))
+  private val varReadOpDiamond = Config(graphType= "diamond", reads = Variable(1,2,1), operations = Variable(4, 10, 3),
```

Comment:
code style: multi-line method call: each arg should be on it's own line - https://docs.scala-lang.org/style/indentation.html#methods-with-numerous-arguments
(The other two calls above should thus also be multi-line for consistency).